ui/src/settings.vue

```diff
@@ -18,7 +18,7 @@
         <b-container v-if="ready">
 
             <!--profile-->
-            <div v-if="tab == 0">
+            <div v-if="tabs[tab].id == 'profile'">
```

Comment:
Do you think we should create a simple computed property like this?

```
computed: {
tabID() {
     return this.tabs[tab].id; 
}
```

And use this in the template?

```
<div v-if="tabID == 'profile'">
```

Maybe it's silly?